core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -154,16 +163,20 @@ class Validator private (private val _name : String, val startStep : Step, val c
   }
 
   private class ValidatorFailGauge(private val timer : Timer,
-                                   private val failMeter : Meter) extends PercentGauge {
-
-    override def getNumerator = failMeter.oneMinuteRate
-    override def getDenominator = timer.oneMinuteRate
+                                   private val failMeter : Meter) extends RatioGauge {
+    override def getRatio: Ratio = Ratio.of(failMeter.oneMinuteRate, timer.oneMinuteRate)
   }
 
-  private val timer = metrics.timer(TIMER_NAME, name)
-  private val failMeter = metrics.meter(FAIL_METER_NAME, FAIL_METER_EVENT, name)
-  private val failGauge =  metricsRegistry.newGauge(getClass, FAIL_RATE_NAME, name,
-                                                   (new ValidatorFailGauge(timer, failMeter)))
+  private val timer = metrics.timer(name, TIMER_NAME)
```

Comment:
The change makes sense, and I appreciate the consistency.

Here's where I'm getting lost: I pulled this from the nl-grons metrics-scala 3.5.2 jar.

```
package nl.grons.metrics.scala
class MetricBuilder(val baseName : nl.grons.metrics.scala.MetricName, val registry : com.codahale.metrics.MetricRegistry) extends scala.AnyRef {
  def gauge[A](name : scala.Predef.String, scope : scala.Predef.String = { /* compiled code */ })(f : => A) : nl.grons.metrics.scala.Gauge[A] = { /* compiled code */ }
  def cachedGauge[A](name : scala.Predef.String, timeout : scala.concurrent.duration.FiniteDuration, scope : scala.Predef.String = { /* compiled code */ })(f : => A) : nl.grons.metrics.scala.Gauge[A] = { /* compiled code */ }
  def counter(name : scala.Predef.String, scope : scala.Predef.String = { /* compiled code */ }) : nl.grons.metrics.scala.Counter = { /* compiled code */ }
  def histogram(name : scala.Predef.String, scope : scala.Predef.String = { /* compiled code */ }) : nl.grons.metrics.scala.Histogram = { /* compiled code */ }
  def meter(name : scala.Predef.String, scope : scala.Predef.String = { /* compiled code */ }) : nl.grons.metrics.scala.Meter = { /* compiled code */ }
  def timer(name : scala.Predef.String, scope : scala.Predef.String = { /* compiled code */ }) : nl.grons.metrics.scala.Timer = { /* compiled code */ }
  protected def metricNameFor(name : scala.Predef.String, scope : scala.Predef.String = { /* compiled code */ }) : scala.Predef.String = { /* compiled code */ }
}
```

It looks to me like the parameters are not varargs anymore, and that MetricRegistry.name(...) would work as the parameter to the timer method. Am I looking at the wrong thing?